contracts/base/strategies/multi/AaveMaiBalStrategyBase.sol

```diff
@@ -12,31 +12,31 @@
 
 pragma solidity 0.8.4;
 
-import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "./../../../openzeppelin/SafeERC20.sol";
 import "./../../../third_party/IERC20Extended.sol";
-import "./../StrategyBase.sol";
+import "./../ProxyStrategyBase.sol";
 import "./pipelines/LinearPipeline.sol";
 import "../../interface/strategies/IMaiStablecoinPipe.sol";
 import "../../interface/strategies/IAaveMaiBalStrategyBase.sol";
 
 /// @title AAVE->MAI->BAL Multi Strategy
 /// @author belbix, bogdoslav
-contract AaveMaiBalStrategyBase is StrategyBase, LinearPipeline, IAaveMaiBalStrategyBase {
+contract AaveMaiBalStrategyBase is ProxyStrategyBase, LinearPipeline, IAaveMaiBalStrategyBase {
   using SafeERC20 for IERC20;
 
   /// @notice Strategy type for statistical purposes
   string public constant override STRATEGY_NAME = "AaveMaiBalStrategyBase";
   /// @notice Version of the contract
   /// @dev Should be incremented when contract changed
-  string public constant VERSION = "1.2.0";
+  string public constant VERSION = "2.0.0";
   /// @dev Placeholder, for non full buyback need to implement liquidation
   uint256 private constant _BUY_BACK_RATIO = 10000;
```

Comment:
why you didn't change it?

Reply:
Sorry. did not changed what? If you about version - then I guess what it is Major.Minor.Fix, 
and I resolved proxy feature as major change.